ntbs-service/Pages/NotificationSummary.cshtml.cs

```diff
@@ -38,9 +38,12 @@ public async Task<ContentResult> OnGetAsync(string notificationId, bool allowDra
             }
 
             var notification = await _notificationRepository.GetNotificationAsync(parsedId);
+
             if (notification == null || (!allowDraft && !notification.HasBeenNotified))
             {
-                return CreateJsonResponse(new { validationMessage = ValidationMessages.IdDoesNotMatchNtbsRecord });
+                return allowLegacyNotifications 
+                    ? CreateJsonResponse(new { warningMessage = "Details not available in NTBS" }) 
+                    : CreateJsonResponse(new { validationMessage = ValidationMessages.IdDoesNotMatchNtbsRecord });
```

Comment:
It's odd to have the messsage copy inline here - it probably just wants to be in ValidationMessages too